repository/src/main/java/org/alfresco/repo/content/ContentServiceImpl.java

```diff
@@ -615,14 +625,25 @@ public boolean isContentDirectUrlEnabled(NodeRef nodeRef)
     /**
      * {@inheritDoc}
      */
+    @Override
+    @Deprecated
     public DirectAccessUrl requestContentDirectUrl(NodeRef nodeRef, boolean attachment, Long validFor)
+    {
+        return requestContentDirectUrl(nodeRef, ContentModel.PROP_CONTENT, attachment, validFor);
+    }
```

Comment:
As above.

Reply:
This should be resolved now but please double check it is what you meant